docs/install/collecting-metrics/collector.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: otel-collector-config
+  namespace: metrics
+data:
+  collector.yaml: |
+    receivers:
+      opencensus:
+        endpoint: "0.0.0.0:55678"
+
+    exporters:
+      logging:
+      prometheus:
+        endpoint: "0.0.0.0:8889"
+
+    service:
+      extensions:
```

Comment:
Are these extensions leverage by knative? Do we required them define here?

Reply:
Net mind my comment these are collector aspects

https://github.com/open-telemetry/opentelemetry-collector/blob/master/extension/README.md

And they document the default ports for health and zpages

Reply:
They aren't strictly necessary, but I found zpages to be useful.